verification/verify_login.py

```
@@ -0,0 +1,33 @@
+import time
+
+def verify_login_screen(page: Page):
+    print("Navigating to http://localhost:3000...")
+    page.goto("http://localhost:3000")
+
+    print("Waiting for 'Welcome Back' text...")
+    # Increase timeout for Flutter initialization
+    try:
+        expect(page.get_by_text("Welcome Back")).to_be_visible(timeout=30000)
+        print("'Welcome Back' found.")
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)